crates/symbolicator-service/src/services/derived.rs

```diff
@@ -63,7 +85,8 @@ where
         &handle.uri(),
         ObjectUseInfo::from_derived_status(
             &cache_entry_as_cache_status(&derived_cache),
-            handle.status(),
+            // FIXME(swatinem): figure out what to do here?
+            &CacheStatus::Positive,
```

Comment:
I think this is correct. `ObjectMetaHandle` is only created in `FetchFileMetaRequest::load`; previously this method had a `status` parameter and that status was inherited by the `ObjectMetaHandle`. As of this PR the status implicitly always positive, so using `Positive` here seems appropriate.